### ISSUE TITLE
Dataset labeling

### ISSUE THREAD

**Issue by Kimmm-pzl** [mui-neuro/swi-cnn#2]
Hello

I would like to ask has your paper published the labeled training set? If not, is it necessary to download public datasets and manually label them to train a model in this field? Through simple thesis research, I would like to ask is it that there are very few labeled datasets published in this field?

Thanks again for your reply!

Best
Kim

**Comment by vbeliveau** [comment]
Unfortunately, data sharing was not included as part of the consent form used to obtain the training data, hence it cannot be shared, sorry. To train a new models you would obviously need to manually label a new dataset, but you can certainly use the existing models to label your own SW images.

**Comment by Kimmm-pzl** [comment]
> Unfortunately, data sharing was not included as part of the consent form used to obtain the training data, hence it cannot be shared, sorry. To train a new models you would obviously need to manually label a new dataset, but you can certainly use the existing models to label your own SW images.

Hi,

Thanks for your clear reply!

I would like to know what software or algorithm you use for manually label your training dataset?

Best
Kim

**Comment by vbeliveau** [comment]
I used itk-snap (http://www.itksnap.org). I strongly recommend it for manual labeling tasks.
